modules/system/lang/zh-tw/client.php

```diff
@@ -0,0 +1,102 @@
+    |
+    */
+
+    'markdowneditor' => [
+        'formatting' => '格式',
+        'quote' => '引用',
+        'code' => '程式碼',
+        'header1' => '標題一',
+        'header2' => '標題二',
+        'header3' => '標題三',
+        'header4' => '標題四',
+        'header5' => '標題五',
+        'header6' => '標題六',
+        'bold' => '粗體',
+        'italic' => '斜體',
+        'unorderedlist' => '項目清單',
+        'orderedlist' => '數字清單',
+        'video' => '影片',
+        'image' => '圖片',
+        'link' => '連結',
+        'horizontalrule' => '插入水平線',
+        'fullscreen' => '全螢幕',
+        'preview' => '預覽',
+    ],
+    'mediamanager' => [
+        'insert_link' => '插入媒體櫃連結',
+        'insert_image' => '插入媒體櫃圖片',
+        'insert_video' => '插入媒體櫃影片',
+        'insert_audio' => '插入媒體櫃音訊',
+        'invalid_file_empty_insert' => '請選擇檔案以插入連結。',
+        'invalid_file_single_insert' => '請選擇一個檔案。',
+        'invalid_image_empty_insert' => '請選擇插入的圖片。',
+        'invalid_video_empty_insert' => '請選擇插入的影片。',
+        'invalid_audio_empty_insert' => '請選擇插入的音訊。',
+    ],
+    'alert' => [
+        'confirm_button_text' => '確認',
+        'cancel_button_text' => '取消',
+        'widget_remove_confirm' => '確定移除此元件？',
+    ],
+    'datepicker' => [
+        'previousMonth' => '上個月',
+        'nextMonth' => '下個月',
+        'months' => ['一月', '二月', '三月', '四月', '五月', '六月', '七月', '八月', '九月', '十月', '十一月', '十二月'],
+        'weekdays' => ['星期日', '星期一', '星期二', '星期三', '星期四', '星期五', '星期六'],
+        'weekdaysShort' => ['週日', '週一', '週二', '週三', '週四', '週五', '週六']
+    ],
+    'colorpicker' => [
+        'choose' => '確定',
+    ],
+    'filter' => [
+        'group' => [
+            'all' => '全部',
+        ],
+        'scopes' => [
+            'apply_button_text' => '確定',
+            'clear_button_text' => '清除',
+        ],
+        'dates' => [
+            'all' => '全部',
+            'filter_button_text' => '篩選',
+            'reset_button_text' => '重置',
+            'date_placeholder' => '日期',
+            'after_placeholder' => '在此之後',
+            'before_placeholder' => '在此之前',
+        ],
+        'numbers' => [
+            'all' => '全部',
+            'filter_button_text' => '篩選',
+            'reset_button_text' => '重置',
+            'min_placeholder' => '最小值',
+            'max_placeholder' => '最大值',
+        ],
+    ],
+    'eventlog' => [
+        'show_stacktrace' => 'Show the stacktrace',
+        'hide_stacktrace' => 'Hide the stacktrace',
+        'tabs' => [
+            'formatted' => 'Formatted',
+            'raw' => 'Raw',
+        ],
+        'editor' => [
+            'title' => 'Source code editor',
+            'description' => 'Your operating system should be configured to listen to one of these URL schemes.',
+            'openWith' => 'Open with',
+            'remember_choice' => 'Remember selected option for this session',
+            'open' => 'Open',
+            'cancel' => 'Cancel',
+        ],
+    ],
```

Comment:
Untranslated strings shouldn't be needed in the client.php file either, I believe it should fallback to the application's fallback locale